src/TestUtils/src/UITest.Appium/Actions/AppiumGeneralActions.cs

```diff
@@ -59,5 +68,53 @@ CommandResponse GetAttribute(IDictionary<string, object> parameters)
 			}
 			return CommandResponse.FailedEmptyResponse;
 		}
+
+		CommandResponse GetSelected(IDictionary<string, object> parameters)
+		{
+			var element = parameters["element"];
+
+			if (element is AppiumElement appiumElement)
+			{
+				return new CommandResponse(appiumElement, CommandResponseResult.Success);
+			}
+			else if (element is AppiumDriverElement driverElement)
+			{
+				return new CommandResponse(driverElement.AppiumElement.Selected, CommandResponseResult.Success);
+			}
+
+			return CommandResponse.FailedEmptyResponse;
+		}
+
+		CommandResponse GetDisplayed(IDictionary<string, object> parameters)
+		{
+			var element = parameters["element"];
+
+			if (element is AppiumElement appiumElement)
+			{
+				return new CommandResponse(appiumElement, CommandResponseResult.Success);
```

Comment:
Is this for older Appium versions? This doesn't seem to check anything specific to displayed, does it? The same of the other ones in here: Enabled, etc.

Reply:
Yes, in all our ui tests using Appium we must have access to the used driver. It a fallback.